tests/test_jira.py

```
@@ -0,0 +1,477 @@
+"""Tests for Jira integration."""
+
+import os
+from typing import Generator
```

Ruff (0.15.0)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)